setup_eunit.template

```diff
@@ -1,6 +1,6 @@
 {variables, [
     {package_author_name, "The Apache Software Foundation"},
-    {cluster_port, 5984},
+    {cluster_port, 0},
```

Comment:
Those config tests needs to be fixed - that's  simple. Otherwise you make half-fixed problem here: eunit tests still may fail because some other app may use 5986 port eventually.